default_config.yaml

```diff
@@ -98,3 +98,5 @@ checkpoint_file_path: "logs/checkpoints"
 # The size to which images should be padded/cropped using PIL.ImageOps.pad
 final_image_size: 500
 
+# The kind of transformation applied: valid options are "default", "random_rotations", "random_transforms"
```

Comment:
The name "default" does not tell at all whats going on. Further how do we combine these transformations here?